pkgs/by-name/pa/parpd/package.nix

```
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "parpd";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "NetworkConfiguration";
+    repo = "parpd";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-6qmoAW9jm7xMRHZUMQLpe0N+UeVnQP8dC4+Iq+d5Eaw=";
+  };
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    homepage = "https://roy.marples.name/projects/parpd";
+    changelog = "https://github.com/NetworkConfiguration/parpd/releases/tag/v${finalAttrs.version}";
+    description = "Proxy ARP Daemon that complies with RFC 1027";
+    maintainers = with lib.maintainers; [ bot-wxt1221 ];
+    platforms = lib.platforms.unix;
```

```suggestion
    platforms = lib.platforms.linux;
```

Looks like it depends on linux features? https://github.com/NetworkConfiguration/parpd/tree/master/src